src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -47,6 +49,9 @@ public Book findByAuthor(@PathVariable String author) {
 	@PostMapping
 	@ResponseStatus(HttpStatus.CREATED)
 	public Book create(@RequestBody Book book) {
+		if(bookRepository.findByAuthor(book.getIsbn()).isPresent()){
```

Comment:
The author is the author name. The ISBN is a code, no the author name. If you want to search if the book is already present, you should do it by ISBN or by Author, not using one and not the other.